docs/trust/accessibility.de.md

```diff
@@ -0,0 +1,111 @@
+# Barrierefreiheit
+
+Die Barrierefreiheit Ihres pretix-Ticketshops ist nicht nur aus ethisch und unternehmerisch von Bedeutung, sondern auch eine rechtliche Verpflichtung.
+Ab 2025 legen der [European Accessibility Act] (https://en.wikipedia.org/wiki/European_Accessibility_Act) und seine nationalen Ausführungsgesetze fest, dass alle Websites, über die Verträge mit Verbraucher*innen geschlossen werden, bestimmten Barrierefreiheitsstandards entsprechen müssen.
```

Comment:
Wollen wir in der deutschen Fassung direkt aufs deutsche und österreichische Recht verlinken?

https://de.wikipedia.org/wiki/Barrierefreiheitsst%C3%A4rkungsgesetz bzw https://www.ris.bka.gv.at/eli/bgbl/i/2023/76/P0/NOR40254410